### PR TITLE
Update the game over message

### PR DESCRIPTION
**Pull request by zexusphere**
Update the game over message so people know how to play again!

### CODE REVIEW

**Comment by sans-hush** [comment]
Good to go merge main deployment

**Comment by zexusphere** [comment]
merge will be ready shortly

**Review by zexusphere — COMMENTED** [review]
just pull req about "added the update the game over message branch". So far, this branch is good to go for merging

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Comment (zexusphere) [inline index.html]:
```suggestion demo for myself ++
<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2> 
```

**Pull request merged**